okhttp/src/jvmMain/kotlin/okhttp3/internal/ws/RealWebSocket.kt

```diff
@@ -63,7 +63,8 @@ class RealWebSocket(
    */
   private var extensions: WebSocketExtensions?,
   /** If compression is negotiated, outbound messages of this size and larger will be compressed. */
-  private var minimumDeflateSize: Long
+  private var minimumDeflateSize: Long,
+  private var maxQueueSize: Long? = DEFAULT_MAX_QUEUE_SIZE
```

Comment:
Why nullable Long??

Reply:
Thanks for reviewing.

Mistake on my part. Im still coding at the moment.